### PR TITLE
Created new example using Vercel SDK and Ollama

### PR DESCRIPTION
**Pull request by mhioi**
HI
It's MH
Hope you great team of Mediar-AI are doing well
Just copied vercel-ai folder and then pasted it by the name vercel-ai-ollama folder, and included all same files as the vercel ai, BUT the actions.tsx; which has been changed for handling ollama responses.
Also changed the md file in mentioned folder.
Sorry for any dirty code or bad PR etc,it is my first PR. Tried my best.
Thanks

### CODE REVIEW

File: examples/typescript/vercel-ai-chatbot-ollama/lib/chat/actions.tsx [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+      }
+    ]
+  })
+
+  let textStream: undefined | ReturnType<typeof createStreamableValue<string>>
+  let textNode: undefined | React.ReactNode
+
+  // Stream UI for the AI response
+  const result = await streamUI({
+    model: ollama('mannix/llama3.1-8b-lexi:tools-q8_0'), // Use Ollama model
```
Comment (louis030195) [inline examples/typescript/vercel-ai-chatbot-ollama/lib/chat/actions.tsx:65]: @mhioi 
cool!

any chance you could let the user specify the model in the UI? somehow

Comment (mhioi) [reply examples/typescript/vercel-ai-chatbot-ollama/lib/chat/actions.tsx:65]: I think that could be done, I'll try my best.

I was thinking about adding a feature to let user see his screen records or screen frames in UI.

Comment (louis030195) [reply examples/typescript/vercel-ai-chatbot-ollama/lib/chat/actions.tsx:65]: just do a free from input and below write "only works with models that supports tools"

Comment (mhioi) [reply examples/typescript/vercel-ai-chatbot-ollama/lib/chat/actions.tsx:65]: ok, I will leave a free text input to let user specify the model.

**Pull request closed without merging**